apps/web/src/modules/create-proposal/components/ReviewProposalForm/ReviewProposalForm.tsx

```diff
@@ -104,6 +105,7 @@ export const ReviewProposalForm = ({
             })
             .then((res) => res.data)
         } catch (err) {
+          console.log('err', err)
```

Comment:
Do we want to sentry log this?

Reply:
Not sure. Would need to learn more about how we use Sentry. So far I've seen it used in the API routes.  

More than happy to add it in. Would also love to get access to Sentry logs so I can see what's not working and which parts of the app can be fixed.